scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -619,10 +619,6 @@ public TaskResult getTaskResultFromIncarnation(JobId jobId, String taskName, int
 
         jlogger.debug(jobId, "trying to get the task result, incarnation " + inc);
 
-        if (inc < 0) {
```

Comment:
I see that if check was duplicated. But maybe you can put jlogger.debug line before check?

Reply:
It would change behaviour. I will not do it

Reply:
Same remark